modules/mongodb/templates/mongodb-backup-s3.erb

```diff
@@ -17,65 +17,109 @@ date_format = '%Y-%m-%d:%H:%M:%S'
 mongo_server = '<%= @server %>'
 key_fingerprint = '<%= @private_gpg_key_fingerprint %>'
 s3_bucket = '<%= @s3_bucket %>'
-standalone = '<%= @standalone %>'
+<% if standalone -%>
```

Comment:
What you're doing is dynamically *re-writing* the script to be different based on your environment values. Telling it to do different things for each environment should be done via command line options which would be based on your environment. Environment would still be handled by hiera. In general do you re-write many apps when you change environment?

I'll agree with it being consistent but I don't think it's correct.